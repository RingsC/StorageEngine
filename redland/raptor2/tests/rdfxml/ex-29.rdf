<?xml version="1.0" encoding="UTF-8"?> 
<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
         xmlns="http://example.org/ns#">

  <!-- Test xml:lang info gets passed to output -->

  <rdf:Description rdf:about="http://example.org/node">
    <prop1 xml:lang="en">chat</prop1>
    <prop2 xml:lang="fr">chat</prop2>
    <prop3>chat</prop3>

  </rdf:Description>

</rdf:RDF>
