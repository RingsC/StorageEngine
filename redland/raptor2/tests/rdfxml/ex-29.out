<http://example.org/node> <http://example.org/ns#prop1> "chat"@en .
<http://example.org/node> <http://example.org/ns#prop2> "chat"@fr .
<http://example.org/node> <http://example.org/ns#prop3> "chat" .
